### PR TITLE
Edited contents in Palash Nigam's profile

### PR DESCRIPTION
**Pull request by itsmepiyush2**

### CODE REVIEW

**Comment by itsmepiyush2** [comment]
Sorry I'm still a newbie on GitHub. I did not understand what you said. Can you please elaborate so that I can make necessary changes?

**Comment by knrt10** [comment]
@itsmepiyush2 please follow these steps:-
1) git checkout master
2) git branch -D Changes
3) git remote add upstream https://github.com/p-society/p-society.github.io.git
4) git rebase upstream/master
5) git checkout -b changes
6) Copy changes for github on 2 files you made 
7) git add -A
8) git commit -m "commit message you want"
9) git push master

**Comment by knrt10** [comment]
After that make a new PR

**Comment by palash25** [comment]
> After that make a new PR

why do you want a new PR ? @knrt10  can't he just amend the commit ? I want @itsmepiyush2 to learn amend and force push

**Comment by palash25** [comment]
Just to make things clear @knrt10 we will always amend commits don't let the juniors take the easy way out :stuck_out_tongue_winking_eye:  if we train them well here and be strict with our reviews it will be easier for them to contribute to major OSS organizations.

File: _contributors-app/palash25.md [hunk 1]
```diff
@@ -50,11 +50,10 @@ project:
      written using Requests and BeautifulSoup.
 
 volunteer_experience: |
- - Most of my time is spent hacking on open source. I have contributed to coala, duckduckgo, Kinto, OpenGenus,Mozilla and the Ethereum foundation.
+ - I am the head of the Programming Society and the club captain of the Mozilla Campus Club of IIIT-Bh. Together we try to guide the first year students in getting started with software development.
 
- - I also like to write articles and have written for opensource.com and geeksforgeeks.org in the past. I also maintain a list of links to all my articles just in case you are interested.
+ - Most of my time is spent hacking on open source. I have made contributions to coala, duckduckgo, Kinto, OpenGenus, Mozilla and the Ethereum foundation. Moreover, I am fond of writing articles. I have prior experience in content writing for opensource.com and geeksforgeeks.org. For those interested, I maintain a list of links to all my articles.
```
Comment (palash25) [inline _contributors-app/palash25.md]: can you add this as a link to the last word of this point i.e `[articles](https://github.com/palash25/articles/)` ?

Comment (palash25) [reply _contributors-app/palash25.md]: The rest LGTM :+1:

Comment (itsmepiyush2) [reply _contributors-app/palash25.md]: Could you tell me what the "remote name" is? I'll do a force push.

Comment (debck) [reply _contributors-app/palash25.md]: The remote repo is the original repo -- https://github.com/p-society/p-society.github.io.git

Comment (palash25) [reply _contributors-app/palash25.md]: Could you please google that first and then ask me if you still don't understand?

**Comment by palash25** [comment]
@knrt10 this wasn't supposed to be merged yet please read the ongoing conversations before merging stuff. Now if I revert this it shows a that I have authored the commit. How do you plan to fix this?

I had also asked @itsmepiyush2 to force push https://github.com/p-society/p-society.github.io/pull/58#discussion_r211093513 how is he supposed to do that now?

File: _contributors-app/palash25.md (hunk 1 above)
Comment (itsmepiyush2) [reply _contributors-app/palash25.md]: Sorry, got it!

Comment (itsmepiyush2) [reply _contributors-app/palash25.md]: I actually did. I couldn't understand so I asked. Sorry.

Comment (palash25) [reply _contributors-app/palash25.md]: remote_name is the name given to the repository url. Do `git remote -v` and tell me what you see.

Comment (itsmepiyush2) [reply _contributors-app/palash25.md]: origin	https://github.com/itsmepiyush2/p-society.github.io.git (fetch)
origin	https://github.com/itsmepiyush2/p-society.github.io.git (push)

Comment (palash25) [reply _contributors-app/palash25.md]: that means origin is your remote name. You will also need to add an upstream remote but we can do that after this PR. The upstream will point to this repo i.e. `p-society/p-society.githu.....`

**Comment by palash25** [comment]
@knrt10 I need you to fix this mess

**Pull request merged**